### PR TITLE
Better handle position aesthetics in params

### PR DESCRIPTION
**Pull request by thomasp85**
revdepchecks uncovered an edge case in direction detection if position aesthetics are given outside aes(). This PR correctly looks in both data and params when figuring out direction

### CODE REVIEW

File: R/geom-linerange.r [hunk 1]
```diff
@@ -99,7 +99,7 @@ GeomLinerange <- ggproto("GeomLinerange", Geom,
 
   setup_params = function(data, params) {
     params$flipped_aes <- has_flipped_aes(data, params, range_is_orthogonal = TRUE)
-    if (!(params$flipped_aes || all(c("x", "ymin", "ymax") %in% names(data)))) {
+    if (!(params$flipped_aes || all(c("x", "ymin", "ymax") %in% c(names(data), names(params))))) {
```
Comment (clauswilke) [inline R/geom-linerange.r]: Just wondering why it tests for `x`, `ymin`, `ymax` but not `y`, `xmin`, `xmax` in `params`. Is this correct?

Comment (thomasp85) [reply R/geom-linerange.r]: if `has_flipped_aes()` returns true it means that it contains y, xmin, and xmax

Comment (clauswilke) [reply R/geom-linerange.r]: Ah, Ok, maybe add a quick comment.

File: R/geom-linerange.r [hunk 2]
```diff
@@ -113,6 +113,7 @@ GeomLinerange <- ggproto("GeomLinerange", Geom,
   },
 
   draw_panel = function(data, panel_params, coord, flipped_aes = FALSE) {
+    browser()
```
Comment (clauswilke) [inline R/geom-linerange.r]: This looks left over. That's probably why the CI fails.

Comment (thomasp85) [reply R/geom-linerange.r]: Oh shit — thanks😅

**Pull request merged**